web_src/css/helpers.css

```diff
@@ -82,9 +82,6 @@ Gitea's private styles use `g-` prefix.
 .gt-w-screen { width: 100vw !important; }
 .gt-h-screen { height: 100vh !important; }
 
-.gt-min-w-0 { min-width: 0 !important; }
-.gt-min-w-16 { min-width: 16px !important; } /*There are some weird bugs with flexboxes and SVGs where the svgs shrink into oblivion*/
```

Comment:
Do these bugs no longer occur for you?
Did you test this when there are lots of branches/tags?
That's when this bug occurs the most.

Reply:
But it's still a flexbox with SVGs?

Reply:
Yep, I cannot reproduce the bug either anymore.
Fine by me.
I still have no idea what caused it or why it isn't present anymore, but if it works, it works…